lib/lintIgnore.js

```diff
@@ -0,0 +1,56 @@
+var merge = require('merge');
+
+module.exports = function(tree, options) {
+	var passes = [];
+	var failures = [];
+	var resolveRelevance = function (feature) {
+		return (!feature.relevant || feature.relevant === true || feature.relevant());
+	};
+	var files = {};
+	files = merge(files, options.ignore);
```

Comment:
`ignore` is an object like so:
```js
ignore: {
 'No debug folder': {
  path: /^debug/i,
  shouldExist: false,
  type: 'tree'
 }
}
```
I use the keys of the ignore object, so having to go through `options` seems like an unneeded step